pylace/README.md

```diff
@@ -44,3 +44,23 @@ engine.update(10_000)
 engine.predict('Class_of_Orbit', given={'Period_minutes': 1436.0})
 # ('GEO', 0.13583714831550336)
 ```
+
+## Tests
+
+To run tests, use `pytest`
+
+```console
+$ pytest -x
+```
+
+To run doctets:
+
+```console
+$ python tests/test_docs.py
```

Comment:
I'm verifying this myself now but it looks like these tests didn't get run in the CI, do we have to add this step to the `python-build-test.yaml` file?

Reply:
Upon further review,
1. It doesn't appear that these doctests are being run in CI
2. I think it's possible that the doctests were never being run in CI?

Reply:
I don't think the doc tests did ever get run or the recent breakage would've been caught. I added the `LACE_DOCTEST_NOPLOT` env var so that we could test it in CI going forward.

Reply:
Recording for future reference: Bax and I discussed it and we think it's okay to not have these tests in CI for the time being